src/apps/users/fixtures/users.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "table": "users",
+    "fields": {
+      "id": 1,
+      "email": "qwe@gmail.com",
+      "full_name": "Tom Isaak",
+      "hashed_password": "Password hashed"
+    }
+  }
+]
```

Comment:
Why do we need these json files in general?
I propose using this library - https://pypi.org/project/pydantic-factories/

```python
from pydantic_factories import ModelFactory

class UserFactory(ModelFactory):
    __model__ = User

# You can pass any fields you need
# It also supports nesting 
john = UserFactory.build(...)
```

Reply:
Hard to manage database state when test need more records